src/expiring_hash_map/tests.rs

```diff
@@ -0,0 +1,92 @@
+use super::*;
+use crate::runtime::Runtime;
+use futures::future::poll_fn;
+use futures_test::task::{new_count_waker, noop_context, panic_context};
```

Comment:
We should use `tokio-test` here plus we can upstream things much easier.

Reply:
`tokio-test` and `futures-test` don't have feature-parity and are not swappable. I'm not sure if it's possible to use `tokio-test` with the same elegance.

Reply:
Nvm, it works pretty well. The approach is different, but I can implement the same assertions. :+1: